src/routers/project/projects.py

```diff
@@ -98,8 +98,125 @@ async def add_collaborator(request: Request,
+                            project_id: int,
+                            db: Session = Depends(get_db),
+                            project_handler: object = Depends(createHandler)):
+    # check project exists
+    project_handler.check_project_exists(project_id, db)
+    # check privileges
+    owned = request.state.owned
+    participating = request.state.participating
+    check_privilege(project_id=project_id,
+                    owned_projects=owned,
+                    participating_projects=participating)
+    return project_handler.get_docs(project_id=project_id, db=db)
+    
+
+@project_router.put("/project/{project_id}/logo", response_model=ProjectLogo)
+async def upload_project_logo(request: Request,
+                              project_id: int,
+                              logo: UploadFile,
+                              db: Session = Depends(get_db),
+                              project_handler: object = Depends(createHandler)):
+    # check project exists
+    project_handler.check_project_exists(project_id, db)
+    # check privilege
+    owned = request.state.owned
+    participating = request.state.participating
+    check_privilege(project_id=project_id,
+                    owned_projects=owned,
+                    participating_projects=participating)
+    # check valid upload file (content_type is image/png or image/jpeg)
+    if logo.content_type not in ["image/png", "image/jpeg"]:
+        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST,
+                            detail="Logo must be a .png or .jpeg file")
+    # call method and return created logo
+    username = request.state.username
+    content = await logo.read()
+    return project_handler.upload_logo(project_id=project_id,
+                                       logo_name=logo.filename,
+                                       b_content=content,
+                                       logo_poster=username,
+                                       db=db)
+    
+
+@project_router.get("/project/{project_id}/logo", response_model=ProjectLogo)
+async def download_logo(request: Request,
+                        project_id: int,
+                        db: Session = Depends(get_db),
+                        project_handler: object = Depends(createHandler)):
+    # checks
+    project_handler.check_project_exists(project_id, db)
+    owned = request.state.owned
+    participating = request.state.participating
+    check_privilege(project_id=project_id,
+                    owned_projects=owned,
+                    participating_projects=participating)
+    # calling method
+    name, content = project_handler.download_logo(project_id=project_id,
+                                                      db=db)
+    return Response(
+            content=content,
+            headers={
+                "Content-Disposition": f"attachment;filename={name}",
+                "Content-Type": "application/octet-stream"
+            }
+        )
+    
+

```

Comment:
Btw, it is not obvious from the first glance, so please check that if you delete the project, all the documents are deleted as well